v2/concordium/types.proto

```diff
@@ -692,9 +702,227 @@ message ConsensusInfo {
 }
 
 // Information about an arrived block that is part of the streaming response.
+message ArrivedBlockInfo {
+  // Hash of the block.
+  BlockHash hash = 1;
+  // Absolute height of the block, height 0 is the genesis block.
+  AbsoluteBlockHeight height = 2;
+}
+
+// The response for GetCryptographicParameters.
+message CryptographicParameters {
+  // A free-form string used to distinguish between different chains even if they share other parameters.
+  string genesis_string = 1;
+  // Generators for the bulletproofs.
+  bytes bulletproof_generators = 2;
+  // A shared commitment key known to the chain and the account holder (and therefore it is public).
+  // The account holder uses this commitment key to generate commitments to values in the attribute list.
+  bytes on_chain_commitment_key = 3;
```

Comment:
We could specify  that this is a pair of group elements (in the same curve as above).